src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/EccTestData.cs

```diff
@@ -45,7 +45,7 @@ internal static IEnumerable<EccTestData> EnumerateApplicableTests()
             CurveOid = Secp256r1OidValue,
             CurveEncodedOidHex = Secp256r1OidHexValue,
 
-            // Suite B Implementers Guide to FIPS 186-3,
+            // Suite B Implementers Guide to FIPS 186-3,
```

Comment:
I'm curious what letter here was weird.  Was this a homoglyph or something?

Reply:
It's value 146.

Reply:
Looks like a custom back-tick within the "private use" range. Outlook shows it fine :)

![image](https://user-images.githubusercontent.com/6527137/63069605-68a8a780-becc-11e9-932b-4948cad113f5.png)

https://www.fileformat.info/info/unicode/char/0092/index.htm

Reply:
Well, that explains why it looks like no change on the web :smile:

Reply:
And VS doesn't show it at all:
![image](https://user-images.githubusercontent.com/2642209/63069720-12e0f900-bee6-11e9-913e-7f30ec845372.png)

This all speaks to why we don't like such characters in the code. :)

Reply:
> Well, that explains why it looks like no change on the web 😄

Which browser? Chrome/Edge show the placeholder/replacement character.

Reply:
I see it like Steve's image in Firefox. (no visible character between r and s)